src/components/Toast/DefaultToast/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import styles from '@/components/Toast/DefaultToast/DefaultToast.css';
+import IconClose from '@/assets/images/icons/web/close.svg';
+
+interface Props {
+  className?: string;
+  autoClose?: number;
+  title?: string;
+  message?: string;
+  onButtonClick: () => void;
```

Comment:
the naming here doesn't really explain the goal of the function. Maybe we could go with an `onClose` which is more clear about when the function will be called